examples/gallery/symbols/basic_symbols.py

```diff
@@ -0,0 +1,116 @@
+"""
+Basic geometric symbols
+-----------------------
+
+The :meth:`pygmt.Figure.plot` method can plot individual geometric symbols
+by passing the corresponding shortcuts to the ``style`` parameter. 
```

Comment:
I think a bit more explanation of the figure example would be helpful here.
```suggestion
by passing the corresponding shortcuts to the ``style`` parameter. The 14 basic
geometric symbols are shown underneath their corresponding shortcut codes.
```